src-tauri/src/remote/static/fonts/urbanist.css

```
@@ -0,0 +1,24 @@
+/* Urbanist (Google Fonts v18), vendored for offline / no-CDN serving.
+ * Weights 400/500/600/700 share the same variable-font woff2 files; we map
+ * each weight to the same source. Served by the embedded remote server at
+ * /static/fonts/*. */
+
+/* latin-ext */
+@font-face {
+  font-family: 'Urbanist';
```

Stylelint (17.12.0)

[error] Expected no quotes around "Urbanist" (font-family-name-quotes)

(font-family-name-quotes)